ConsoleApp/Library/Book.cs

```diff
@@ -0,0 +1,8 @@
+namespace ConsoleApp.Library;
+class Book
+{
+    public int bookId { get; init; }
```

Comment:
code convention violated here, it should be
```suggestion
    public int BookId { get; init; }
```

Reply:
Done!